monkey/monkey_island/cc/ui/src/styles/Collapse.scss

```diff
@@ -1,8 +1,9 @@
 $transition: 300ms cubic-bezier(0.6, 0.3, 0.3, 0.6);
 
 $danger-color: #ebbcba;
+$disabled-color: #b7c2ff;
 $info-color: #ade3eb;
-$default-color: #e0ddde;
```

Comment:
We should keep default so that the report doesn't crash on other people's forks/tests of Monkey